platform-managers/K8sPlatformManager.js

```diff
@@ -1,13 +1,29 @@
 'use strict';
 
 const BasePlatformManager = require('./BasePlatformManager');
+const Promise = require('bluebird');
 
 class K8sPlatformManager extends BasePlatformManager {
   constructor(platform) {
     super(platform);
     this.platform = platform;
   }
 
+  preUnbindOperations(options) {
```

Comment:
Wouldn't it be better to have the default implementations in `BasePlatformManager`? Because I don't see sharing instances as a feature coming from K8S anytime soon